src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,58 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * Handles the System input and output
+ */
+public class Ui {
+    private Scanner scanner;
+    public Ui() {
+        scanner = new Scanner(System.in); // Create a Scanner object
+    }
+
+    public void showWelcome() {
+        String botName = "HuAI";
+        System.out.printf("Hello! I'm %s\nWhat can I do for you?\n", botName);
+    }
+
+    public void showBye() {
+        System.out.println("Bye. Hope to see you again soon!");
+    }
+
+    public String readCommand() {
+        return scanner.nextLine();
+    }
+
+    public void showln(String s) {
```

Comment:
Perhaps naming this method as showLine would be better? I think it would be more consistent with the other method names in this class

Reply:
Yes.